### ISSUE TITLE
my host is not having AVX512BW, What should I do?

### ISSUE THREAD

**Issue by xilu0** [openwall/john#5239]
### Checklist ###
- :1st_place_medal: I've read and understood these instructions;
  - This is not a support forum, it's a bug tracker. For questions and support, review postings on the [john-users mailing list](https://www.openwall.com/lists/john-users/).
- :thumbsup: I've tested using latest bleeding version from this repository.
  - Be clear about your environment and what you are doing. Share a sample hash or file that can be used to reproduce.
- :confused: I'm confused and I need guidance.
  - Please, read the instructions at https://www.openwall.com/john/#lists, then join the list before posting.

### IMPORTANT ###
We expect only reports of issues with the latest revision found in this GitHub repository. We do not expect in here, and have little use for, reports of issues only seen in a release or in a distro package.

Attach details about your OS and about john, including:
- The output of `./john --list=build-info`.
- The command line you are using.

**Comment by ghost** [comment]
Most likely you don't have a CPU that supports AVX512BW. A good starting point to check if your cpu supports avx512 is https://en.wikipedia.org/wiki/AVX-512#CPUs_with_AVX-512

In layman's terms: your Volkswagen Beetle Type 1 does not support hydrogen as a fuel because it does not have a hydrogen-ready engine.

**Comment by solardiz** [comment]
This isn't an issue we need to track, so I'll close it, but please feel free to add comments.

**Comment by xilu0** [comment]
> 

thank you for your answer, so can't my machine use John?

**Comment by xilu0** [comment]
@claudioandre-br 
> Most likely you don't have a CPU that supports AVX512BW. A good starting point to check if your cpu supports avx512 is https://en.wikipedia.org/wiki/AVX-512#CPUs_with_AVX-512
> 
> In layman's terms: your Volkswagen Beetle Type 1 does not support hydrogen as a fuel because it does not have a hydrogen-ready engine.

thank you for your answer, so can't my machine use John?

**Comment by xilu0** [comment]
i get it, i need to compile on the target machine. thank you john
